### ISSUE TITLE
No exports condition for Sveltekit v2 (Deprecation Warning)

### ISSUE THREAD

**Issue by Rar9** [carstenlebek/svelte-email#29]
[vite-plugin-svelte] WARNING: The following packages have a svelte field in their package.json but no exports condition for svelte.

svelte-email@0.0.4

Please see https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/faq.md#missing-exports-condition for details.

**Comment by TonyGibson** [comment]
I've got this also, thanks for creating the request! Should be a quick fix :)

**Comment by Rar9** [comment]
For sveltekit 2 u need to add

"exports": {
".": {
"svelte": "./src/index.js"
}
},

**Comment by FranciscoHS19** [comment]
Is there any news on this fix? I love this package, it is a real time saver, but I'm starting to get a bit worried about using it going forward, as it does not seem like it is being maintained. 

> For sveltekit 2 u need to add
> 
> "exports": { ".": { "svelte": "./src/index.js" } },

@Rar9 I tried to do this fix myself, but I'm getting an error everywhere I'm using this package: "Cannot find module 'svelte-email' or its corresponding type declarations.". Any help would be appreciated.

**Comment by cmjoseph07** [comment]
@FranciscoHS19 
You would not add the export to your project, to fix you would need to fork and make the update in the fork of svelte-email or you would need to do a PR. I have been using this for awhile so I am considering forking it and maintaining it but not sure maintainer is coming back or not.

**Comment by TonyGibson** [comment]
@Rar9 @cmjoseph07 exactly! Hey @carstenlebek love your work! Is adding this code something that is on the list? Would love to keep this repo going to avoid confusion, but if it's not maintained it's best that we fork it and build on the new version that is supported by SK2

**Comment by cmjoseph07** [comment]
@carstenlebek @Rar9 @FranciscoHS19 

Hello all, I have decided to fork the project and maintain it as various people have been asking for updates on the project. I have updated the package and ARIA issues thus far and will see what the community wants in regards to future changes. I hope this is not stepping on the author's project as I love it and have enjoyed using it. If the author returns I would not mind archiving the fork either.

For now, if you want to have these two previously mentioned alerts disappear and work on other issues please check-out:

https://github.com/cmjoseph07/svelty-email

**Comment by Rar9** [comment]
Thanks @cmjoseph07, looking forward for your next version with tailwindcss support.

**Comment by polaroi8d** [comment]
I sent a message to the author to try to get access to maintain and fix the issues, instead of forking the repository.
